bluesky-social/social-app
# remove precacheThreadPostProfiles

## Test Plan

- Press user avatars in the feed
- Press user avatars in post threads

Same behavior of having a minimal shell should exist - the basic info like avatar and display name should be pre-populated on the push.
